frontend/src/components/form/form.js

```diff
@@ -29,74 +35,79 @@ function FlightForm() {
     })
       .then(response => {
         if (!response.ok) {
-          throw new Error('Sorry, an error has occured. Please try again.');
+          throw new Error('Sorry, an error has occurred. Please try again.');
         }
-        return response.json()
+        return response.json();
       })
       .then(data => {
-        if (data && data.flights.items > 0) {
           console.log(data);
           setFlights(data.flights);
           console.log('flights', data);
-        } else {
```

Comment:
Not sure why we're removing this if/else statement? This checks if there are no flights returned and if not it will return an error message? Otherwise, the flight data comes back empty and will mean that the page will just break without this.

Reply:
Ah, yeah sorry I removed it due to it wasn't working correctly as data was getting city data, so it was more than one on the .length. I've fixed it now, so it should give the error message based upon the flights!